configs/stage/roles/content-sources.json

```diff
@@ -0,0 +1,32 @@
+{
+    "roles": [
+      {
+        "name": "Repositories administrator",
+        "description": "Perform any available operation against any repositories resource.",
+        "system": true,
+        "platform_default": false,
+        "admin_default": true,
+        "version": 1,
+        "access": [
+          {
+            "permission": "content-sources:repositories:read"
+          },
+          {
+            "permission": "content-sources:repositories:write"
+          }
+        ]
+      },
+      {
+        "name": "Repositories viewer",
+        "description": "Perform read only operations against repositories resources.",
+        "system": true,
+        "platform_default": true,
+        "version": 2,
```

Comment:
should this be version 1?

Reply:
no, as I understand it version 2 ensures that it propagates to all current users if I understand [this documentation](https://github.com/RedHatInsights/rbac-config#canned-roles) correctly.

Reply:
@coderbydesign ^^

Reply:
This should be fine as is. We need to update the README to reflect that new roles can now start at 1, but starting at 2 works as well! :)